source/guide_prosody.rst

```diff
@@ -0,0 +1,404 @@
++-------------------------------------------------+-------+-------+-------+----------+--------+----------+-------------------------+
+|                isabell.uber.space               |  3600 |   IN  |   A   |          |        |          |      35.195.215.42      |
++-------------------------------------------------+-------+-------+-------+----------+--------+----------+-------------------------+
+|                isabell.uber.space               |  3600 |   IN  |  AAAA |          |        |          |            0:           |
++-------------------------------------------------+-------+-------+-------+----------+--------+----------+-------------------------+
+|            upload.isabell.uber.space            |  3600 |   IN  | CNAME |          |        |          | isabell.uber.space      |
++-------------------------------------------------+-------+-------+-------+----------+--------+----------+-------------------------+
+|             xmpp.isabell.uber.space             |  3600 |   IN  | CNAME |          |        |          | isabell.uber.space      |
++-------------------------------------------------+-------+-------+-------+----------+--------+----------+-------------------------+
+| _xmpp-client._tcp.isabell.uber.space            | 18000 | IN    | SRV   | 0        | 5      | C2S-PORT | xmpp.isabell.uber.space |
++-------------------------------------------------+-------+-------+-------+----------+--------+----------+-------------------------+
+| _xmpp-server._tcp.isabell.uber.space            | 18000 | IN    | SRV   | 0        | 5      | S2S-PORT | xmpp.isabell.uber.space |
++-------------------------------------------------+-------+-------+-------+----------+--------+----------+-------------------------+
+| _xmpp-server._tcp.conference.isabell.uber.space | 18000 | IN    | SRV   | 0        | 5      | S2S-PORT | xmpp.isabell.uber.space |
++-------------------------------------------------+-------+-------+-------+----------+--------+----------+-------------------------+
+
+Configure luarocks
+------------------
+
+Prosody is written in ``lua`` and has some runtime dependencies which we install with the package manager ``luarocks``. In order to run the installed packages we have to adapt the path-variable in ``~/.bash_profile``:
+
+::
+
+ PATH=$HOME/.luarocks/bin:$PATH
+ 
+ export PATH
+
+Additionally we need to provide the paths ``LUA_PATH`` as well as ``LUA_CPATH`` and have to reload to use ``luarocks`` accordingly:
+
+::
+
+ [isabell@stardust ~]$ echo "$(luarocks path)" >> ~/.bash_profile
+ [isabell@stardust ~]$ source ~/.bash_profile
+ [isabell@stardust ~]$
+
+Install runtime lua-dependencies
+--------------------------------
+
+The following dependencies_ (``luasocket``, ``luaexpat``, ``luafilesystem`` and ``luasec``) are required:
+
+::
+
+ [isabell@stardust ~]$ luarocks install luasocket --local
+ luasocket [...] is now built and installed in [...]
+ [isabell@stardust ~]$ luarocks install luaexpat --local
```

Comment:
The server outputs a warning:

> The version of LuaExpat on your system does not support stanza size limits, which may leave servers on untrusted networks (e.g. the internet) vulnerable to denial-of-service attacks. You should upgrade to LuaExpat 1.3.0 or higher as soon as possible. See https://prosody.im/doc/depends#luaexpat for more information.

Is there a way to install the correct version here?

Reply:
The installed version of LuaExpat is in the version 1.3.3-1. You can check it with ```luarocks list```. I have no idea why this is showing up. I'll investigate to solve this issue.

Reply:
Seems to be a known issue -> https://issues.prosody.im/1375. I've succesfully tested the version 1.3.0-1 and will adapt the guide.